aleph_alpha_client/chat.py

```diff
@@ -141,7 +141,15 @@ def to_json(self) -> Mapping[str, Any]:
         payload = {k: v for k, v in asdict(self).items() if v is not None}
         payload["messages"] = [message.to_json() for message in self.messages]
         if self.response_format:
-            payload["response_format"] = self.response_format.to_json()
+            # Handle Pydantic models by converting them to JSONSchema first
+            if hasattr(self.response_format, 'model_json_schema'):
```

Comment:
Once we use `Type[BaseModel]` this becomes `isinstance(response_format, type) and issubclass(response_format, BaseModel)`.